core/cc/android/crash_handler.cpp

```diff
@@ -34,9 +38,24 @@ namespace core {
 
 CrashHandler::CrashHandler()
     : mNextHandlerID(0),
-      mExceptionHandler(new google_breakpad::ExceptionHandler(
-          google_breakpad::MinidumpDescriptor("./crashes"), NULL, ::handleCrash,
-          reinterpret_cast<void*>(this), true, -1)) {
+      mExceptionHandler(nullptr) {
+
+    std::string appInfo;
```

Comment:
Is there not a standard temp directory that we could use?
std::mkdtmp?

Reply:
Since this is Android, just use `/data/local/tmp`

Reply:
Android no more gives permission to write under that folder as far as I can see.

Reply:
if accumulating the crash files a problem, we can delete the crash log after we upload it.

Reply:
Changed it to under cache folder.